android-template/app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,8 @@
             android:name="com.getcapacitor.myapp.MainActivity"
             android:label="@string/title_activity_main"
             android:theme="@style/AppTheme.NoActionBarLaunch"
-            android:launchMode="singleTask">
+            android:launchMode="singleTask"
+            android:exported="true">
```

Comment:
Why is this set to `"true"`?

According to developer.android.com/about/versions/12/behavior-changes-12#exported it should be set to `"false"` in most cases when the app is not a launcher. What is the case for it being `"true"`for Capacitor?

I should mention that my context is making this work for a Capacitor 2 project.